fix terraform clean bugs (#870)
cloudposse/atmos: website/docs/cli/commands/terraform/usage.mdx

- `atmos terraform clean` command deletes the `.terraform` folder, `.terraform.lock.hcl` lock file, and the previously generated `planfile`
  and `varfile` for the specified component and stack. Use the `--skip-lock-file` flag to skip deleting the `.terraform.lock.hcl` file. 
  It deletes all local Terraform state files and directories (including `terraform.tfstate.d/`) for all components and stacks. 
  The `--force` flag bypasses the safety confirmation prompt and forces the deletion. Use with caution.

  :::warning

>   The `clean` performs destructive operations that can lead to permanent state loss. Always ensure you have remote state configured in your components before proceeding.

Reviewer: ```suggestion
  The `clean` command performs destructive operations that can lead to permanent state loss, if not using remote backends. Always ensure you have remote state configured in your components before proceeding.
```

- `atmos terraform clean` command deletes the `.terraform` folder, `.terraform.lock.hcl` lock file, and the previously generated `planfile`
  and `varfile` for the specified component and stack. Use the `--skip-lock-file` flag to skip deleting the `.terraform.lock.hcl` file. 

>   Use the `--everything` flag is the default behavior to delete all the local Terraform state files and directories (including `terraform.tfstate.d`) for all components and stacks. 

Reviewer: ```suggestion
  The --everything flag is the default behavior. It deletes all local Terraform state files and directories (including terraform.tfstate.d) for all components and stacks. 
```

>   Use the `--force` flag to bypass the safety confirmation prompt and force the deletion (use with caution).

Reviewer: ```suggestion
  The --force flag bypasses the safety confirmation prompt and forces the deletion. Use with caution.
```
